ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -462,6 +473,7 @@ PrjFS_Result PrjFS_WriteSymLink(
     
     if (nullptr == relativePath || nullptr == symLinkTarget)
     {
+        LogWarning("PrjFS_WriteSymLink: relativePath=%s or symLinkTarget=%s is null", relativePath, symLinkTarget);
```

Comment:
Similarly to `PrjFS_WritePlaceholderFile`, I'm thinking we should just `assert` that these are non-null?

Reply:
See previous comment.  We try to only exit the process if we are totally hosed.

Reply:
Log these as errors?

Reply:
> Log these as errors?
If we do that we'll get it thru telemetry twice since it's handled as an error farther up.  We can discuss more later today.